src/dra/play.c

```diff
@@ -21,16 +21,278 @@ void func_800E4970(void) {
     HideAllBackgroundLayers();
     func_800EAEEC();
     func_800EDAE4();
-    PlaySfx(0x12);
-    PlaySfx(0xB);
+    PlaySfx(SET_UNK_12);
+    PlaySfx(SET_UNK_0B);
     func_80132760();
     func_800E493C();
 }
 
 void func_800E4A04(void) { s32 pad[3]; }
 
-#if defined(VERSION_US)
-INCLUDE_ASM("asm/us/dra/nonmatchings/play", HandlePlay);
-#elif defined(VERSION_HD)
-INCLUDE_ASM("asm/hd/dra/nonmatchings/play", HandlePlay);
-#endif
+void HandlePlay(void) {
+    s32 i;
+    u8* ptr;
+    u16 *sx, *sy, *ex, *ey;
+    Primitive* prim;
+
+    switch (g_GameStep) {
+    case Play_Reset:
+        D_8003C9A4 = 0;
+        D_8003C738 = 0;
+        if (D_8003C730 == 0) {
+            for (i = 0; i < LEN(D_8003BDEC); i++) {
+                D_8003BDEC[i] = 0;
+            }
+            D_8003BDEC[0xB9] = 1;
+            D_8003BDEC[0x9B] = 1;
+            if (g_CurrentPlayableCharacter != 0) {
+                D_8003BDEC[0x35] = 1;
+                D_8003BDEC[0x62] = 1;
+                D_8003BDEC[0x63] = 1;
+                D_8003BDEC[0x85] = 1;
+                D_8003BDEC[0x95] = 1;
+                D_8003BDEC[0x96] = 1;
+            }
+
+            for (i = 0, ptr = &g_MenuNavigation; i < sizeof(MenuNavigation);
+                 i++) {
+                *ptr++ = 0;
+            }
+
+            for (i = 0; i < BUTTON_COUNT; i++) {
+                g_Settings.buttonConfig[i] = i;
+                g_Settings.buttonMask[i] = D_800A0160[i];
+            }
+            for (i = 0; i < LEN(g_Settings.cloakExteriorColors) +
```

Comment:
I think this suggests there was a single array of cloak colors but the current way we have it is probably more readable. I don't think this needs to be changed, just observing.